rq/group.py

```diff
@@ -50,13 +52,27 @@ def cleanup(self, pipeline: Optional['Pipeline'] = None):
             job_ids = [job for job in job_ids if job not in expired_jobs]
             pipe.srem(self.key, *expired_jobs)
 
-        if not job_ids:
-            self.delete()
         if pipeline is None:
             pipe.execute()
 
+    def enqueue_many(
+        self, queue: Union[str, Queue], job_datas: List['EnqueueData'], pipeline: Optional['Pipeline'] = None
```

Comment:
We can simplify this a bit by only accepting `queue: Queue`, what do you think?

Reply:
Works for me, done